scss/grid/_gutter.scss

```diff
@@ -90,3 +90,37 @@
   @warn 'This mixin is being replaced by grid-col-gutter(). grid-col-uncollapse() will be removed in Foundation 6.4.';
   @include grid-column-uncollapse($gutter);
 }
+
+/// Sets bottom marin on block grid columns to match gutters
+/// @param {Number|Keyword} $margin [auto]
+///   The bottom margin on block grid columns, accepts multiple values:
+///   - A single value will make the margin that exact size.
+///   - A breakpoint name will make the margin the corresponding size in the $gutters map.
+///   - "auto" will make the margin responsive, using the $gutters map values.
```

Comment:
You're right, I copied the technique straight from `grid-column-gutter`. Nice catch. I've fixed the notes.